docs/en/advanced/Scalability-Tuning.md

```diff
@@ -83,6 +80,18 @@ The first one controls the heartbeat intervals for block service in Alluxio and
 filesystem service.
 Again, increase the interval to reduce the number of heartbeat checks.
 
+### Keepalive Time and Timeout
+
+Alluxio worker is configured to check the health of connected clients by sending keepalive pings.
+This is controlled by the following properties
+```properties
+alluxio.worker.network.keepalive.time=30s
+alluxio.worker.network.keepalive.timeout=30s
+```
+The first one controls the maximum wait time since a client sent the last message before worker issues a
```

Comment:
How do you feel about explicitly referencing the property name as opposed to just saying "the first one", "second one"? I think it makes the documentation easier to read and removes any possible ambiguities

Reply:
Updated all property references in this doc.